app/workers/retire_cellect_worker.rb

```diff
@@ -0,0 +1,15 @@
+class RetireCellectWorker
+  include Sidekiq::Worker
+
+  def perform(subject_id, workflow_id)
+    workflow = Workflow.find(workflow_id)
+    if Panoptes.use_cellect?(workflow)
+      smses = workflow.set_member_subjects.where(subject_id: subject_id)
+      smses.each do |sms|
+          Subjects::CellectClient
+          .remove_subject(subject_id, workflow_id, sms.subject_set_id)
+      end
+    end
+  rescue Subjects::CellectClient::ConnectionError, ActiveRecord::RecordNotFound
```

Comment:
Always use explicit requires in Sidekiq workers for non-app constant paths or you'll raise autoload exceptions the first time it's accessed.